backend/services/operations.go

```diff
@@ -283,6 +284,18 @@ func ReadOperation(ctx context.Context, db *database.Connection, operationSlug s
 		userCanViewGroups = true
 	}
 
+	var userIsAdmin bool
+	if err := policyRequireWithAdminBypass(ctx, policy.CanExportOperationData{OperationID: operation.ID}); err == nil {
+		userIsAdmin = true
+	} else {
+		userIsAdmin = middleware.IsAdmin(ctx)
+	}
+
+	var userCanExportData bool
+	if os.Getenv("ENABLE_EVIDENCE_EXPORT") == "true" && userIsAdmin {
```

Comment:
This should probably go in webconfig.go and have a function to retrieve the parsed value.
Also, you could do this:

`userCanExportData := os.Getenv("ENABLE_EVIDENCE_EXPORT") == "true"  && userIsAdmin`

Reply:
ah did not know that existed - thanks for the heads up!